src/core_plugins/kibana/public/management/sections/objects/lib/get_relationships.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash';
```

Comment:
Maybe just

```js
import { get } from 'lodash';
```

We've found it helps to make the code more readable in Monitoring to limit the scope of what we import.